doc/releases/reef.rst

```diff
@@ -5,6 +5,417 @@ Reef
+* rgw: Add missing empty checks to the split string in is_string_in_set() (`pr#56347 <https://github.com/ceph/ceph/pull/56347>`_, Matt Benjamin)
+* rgw: d3n: fix valgrind reported leak related to libaio worker threads (`pr#54852 <https://github.com/ceph/ceph/pull/54852>`_, Mark Kogan)
+* rgw: do not copy olh attributes in versioning suspended bucket (`pr#55606 <https://github.com/ceph/ceph/pull/55606>`_, Juan Zhu)
+* rgw: fix cloud-sync multi-tenancy scenario (`pr#54328 <https://github.com/ceph/ceph/pull/54328>`_, Ionut Balutoiu)
+* rgw: object lock avoids 32-bit truncation of RetainUntilDate (`pr#54674 <https://github.com/ceph/ceph/pull/54674>`_, Casey Bodley)
+* rgw: only buckets with reshardable layouts need to be considered for resharding (`pr#54129 <https://github.com/ceph/ceph/pull/54129>`_, J. Eric Ivancich)
+* RGW: pubsub publish commit with etag populated (`pr#56453 <https://github.com/ceph/ceph/pull/56453>`_, Ali Masarwa)
+* rgw: RGWSI_SysObj_Cache::remove() invalidates after successful delete (`pr#55716 <https://github.com/ceph/ceph/pull/55716>`_, Casey Bodley)
+* rgw: SignatureDoesNotMatch for certain RGW Admin Ops endpoints w/v4 auth (`pr#54791 <https://github.com/ceph/ceph/pull/54791>`_, David.Hall)
+* Snapshot schedule show subvolume path (`pr#56419 <https://github.com/ceph/ceph/pull/56419>`_, Ivo Almeida)
+* src/common/options: Correct typo in rgw.yaml.in (`pr#55445 <https://github.com/ceph/ceph/pull/55445>`_, Anthony D'Atri)
+* src/mount: kernel mount command returning misleading error message (`pr#55300 <https://github.com/ceph/ceph/pull/55300>`_, Neeraj Pratap Singh)
+* src/osd/OSDMap.cc: Fix encoder to produce same bytestream (`pr#55712 <https://github.com/ceph/ceph/pull/55712>`_, Kamoltat)
+* test/libcephfs: skip flaky timestamp assertion on Windows (`pr#54614 <https://github.com/ceph/ceph/pull/54614>`_, Lucian Petrut)
+* test/rgw: increase timeouts in unittest_rgw_dmclock_scheduler (`pr#55790 <https://github.com/ceph/ceph/pull/55790>`_, Casey Bodley)
+* test: explicitly link to ceph-common for some libcephfs tests (`issue#57206 <http://tracker.ceph.com/issues/57206>`_, `pr#53635 <https://github.com/ceph/ceph/pull/53635>`_, Venky Shankar)
+* tools/ceph_objectstore_tool: action_on_all_objects_in_pg to skip pgmeta (`pr#54693 <https://github.com/ceph/ceph/pull/54693>`_, Matan Breizman)
+* Tools/rados: Improve Error Messaging for Object Name Resolution (`pr#55112 <https://github.com/ceph/ceph/pull/55112>`_, Nitzan Mordechai)
+* tools/rbd: make 'children' command support --image-id (`pr#55617 <https://github.com/ceph/ceph/pull/55617>`_, Mykola Golub)
+* use raw_cluster_cmd instead of run_ceph_cmd (`pr#55836 <https://github.com/ceph/ceph/pull/55836>`_, Venky Shankar)
+* v18.2.1 (`pr#54532 <https://github.com/ceph/ceph/pull/54532>`_, Ceph Release Team, Guillaume Abrioux, Igor Fedotov, Venky Shankar, Alexander Mikhalitsyn)
```

Comment:
I hate to sound like a broken record, but the tag of the previous release shouldn't be present in the list of changes of the next release.  If the script gets confused and isn't getting fixed, please make a note to remove a tag manually.

Reply:
removed and pls don't hesitate to sound as you do :)